CMakeLists.txt

```diff
@@ -320,15 +320,14 @@ write_basic_package_version_file(
       COMPATIBILITY SameMajorVersion
       )
 
-
-
 install(TARGETS yaml-cpp 
-    EXPORT "${targets_export_name}"
-    INCLUDES DESTINATION "${include_install_dir}"
-    LIBRARY DESTINATION ${lib_install_dir}
-    ARCHIVE DESTINATION ${lib_install_dir}
-    PUBLIC_HEADER DESTINATION ${include_install_dir}
-    )
+	EXPORT "${targets_export_name}"
+	INCLUDES DESTINATION "${include_install_dir}"
+	RUNTIME DESTINATION ${lib_install_dir}
```

Comment:
Usually another directory (not `lib/`) used here so the `*.exe` and `*.dll` files stored in one directory. You can introduce `bin_install_dir` variable with `bin` value.

Reply:
Is that compatible with Hunter?

Reply:
Should be fine, Hunter doesn't really rely on this part.

Reply:
ok. I saw that the ${LIB_INSTALL_DIR} is configured for such behavior. I will use it for the runtime part.

Reply:
As I already said usually it goes to `bin`, i.e. `BIN_INSTALL_DIR`. Actually `bin` is already used here:
* https://github.com/hunter-packages/yaml-cpp/blob/5ba9e664bc65e4c0e3537d433f33d1fec3bc5e65/CMakeLists.txt#L262-L266

Reply:
So do you prefer that I use directly bin?